### PR TITLE
Create Plugin/Desktop Helper Sample

### PR DESCRIPTION
**Pull request by jake-pauls**
**Overview**
Adds a sample that uses an Electron desktop helper app (React) to communicate with a Photoshop plugin (webpack/vanilla js). 

**Functionality**
The Electron helper app uses an embedded websocket server ([Socket.io](https://socket.io/)) to communicate to and from the UXP helper app. The websockets have the functionality to automatically connect and reconnect with each other. As well, if one of the clients close, the other is notified of the disconnection.

To put a use case into practice, users can send strings to and from the UXP and Electron helper apps. Sent strings are received and displayed in their corresponding places on the UI.

**Closes Issue #13**

### CODE REVIEW

**Comment by kerrishotts** [comment]
I can get both running, but neither seem to see each other. Maybe let's go through how you're running them in case I'm missing a step.

**Comment by jake-pauls** [comment]
Awesome! I just created a video of launching and connecting the UXP plugin/Electron helper. As addressed in the video, the only steps I didn't show was actually cloning the repo and running a quick `yarn install` to grab each of the folder's dependencies.

There's a bit of a cut three minutes in, but I just added that because my computer took around 20 seconds to boot up the electron bundle. Nothing else changed.

Let me know what happens!

[YouTube Link](https://youtu.be/0L8yM3ZU7Ps)

File: desktop-helper-sample/helper/public/server.js [hunk 1]
```diff
@@ -11,8 +11,9 @@ const startServer = async () => {
 
   const io = new Server(server, {
     cors: {
-      origin: 'http://localhost:3000',
+      origin: '*',
       methods: ['GET'],
+      transports: ['websocket'],
```
Comment (jake-pauls) [inline desktop-helper-sample/helper/public/server.js:16]: Hey @kerrishotts, just added some changes to this that (hopefully 🤞 ) enables the Socket.IO connection on Mac. Let me know if this change connects the two properly!

[Solution on CC Developer Forum](https://forums.creativeclouddeveloper.com/t/socket-io-client-doesnt-work/3355/9)

Comment (kerrishotts) [reply desktop-helper-sample/helper/public/server.js:16]: It's alive! :-)

**Pull request merged**